src/book/03-guides/04-general/01-git.mdx

```diff
@@ -64,6 +64,46 @@ If your code is in a different repository and it uses C++ or protobuf files, use
+- **Approve**: this allows the PR to be merged, so don't do this unless you are _reasonably_ confident that the changes are correct.
+- **Comment**: this adds your comments without blocking or approving the changes. Do this if you are not too concerned about the comments you have made, or if you're not sure when you'll be able to approve the PR once the changes are made.
+
+Reviewing code that you may not be fully confident with and leaving comments asking for clarification helps you learn and helps the author write more readable code, which is better for everyone.
+
+## Writing Reviewable Code
+
+Writing reviewable code is a skill that can be developed with practice. Here are some general guidelines to make your PR easier to review:
+
+- **Make it atomic**: A reviewable PR should only address one issue or add one feature.
+- **Keep it small**: Ideally, a reviewable PR should not change much more than 400 lines of code, anything above this super-exponentially increases the time it takes to review, and makes it more likely that an error will be missed.
+- **Name it well**: A reviewable PR should be named starting with a verb saying what you did, what issue was fixed, or what feature added. Prefer the present tense, e.g. "Update dependency X to version Y" instead of "Updated dependency X to version Y".
+
+To keep PRs small and atomic you may have to split your PR into multiple sub PRs. To do this, make a new branch branch from the original PR and move over some code. Then make a separate PR targeting the original PR's branch. Make sure to mention the original PR in the new PR's description, using `#123`, where `123` is the sub PR's number. After completing the sub PR, merge it back into the original PR.
+
+## Handling Review and Reviewee Etiquette
+
+Here are some tips for handling reviews as the author of a PR:
+
+- When a review comment is left on your PR, you may respond with a reply on GitHub, or with a comment in the code. Prefer comments in the code where appropriate, so in the future others don't have to read through several PRs and can instead just read the code.
+- When applying changes, you can go to the **Files Changed** tab and see the comments with the context of the code. Here you can also batch apply suggestions, so they only take one commit.
+- After addressing a review comment, you can click **Resolve Conversation** to hide the conversation thread and reduce visual clutter. It is the reviewer's responsibility to check that your changes adequately addressed their concerns.
+- Don't be alarmed when a reviewer makes over a dozen comments on your PR, just work through them systematically, and don't hesitate to ask for help or clarification if you need more details.
+- Finally, when all the comments have been addressed from a review that requested changes, you can request another review by clicking the **Re-request review** button next to the reviewer's name.
+
 ## Work in Progress PRs
```

Comment:
This section now looks a bit out of place at the bottom here. I'd move it to after **Formatting and Buildkite Checks**.